frontend/src/components/auth/ResetPassword.tsx

```diff
@@ -49,6 +49,7 @@ const ResetPassword = ({ email }: ResetPasswordProps) => {
       if (typeof err === "string") {
         handleErrorOnReset(err);
       } else {
+        // eslint-disable-next-line no-console
         console.log(err);
```

Comment:
oh shoot, we should really window.alert the errs so the user can see them. I'll make a note to swap them all in the code base
where you dealing with eslint errors? Is that why theses have been added in?

Reply:
Sorry should've added a comment about this; the firebase build kept failing with an error saying `Treating warnings as errors because process.env.CI = true`
https://github.com/uwblueprint/planet-read/runs/4676380763?check_suite_focus=true